components/ui/__tests__/Alert-test.tsx

```diff
@@ -17,29 +17,36 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-
-// TODO : https://github.com/styled-components/jest-styled-components#global-installation
-import 'jest-styled-components';
+import { shallow } from 'enzyme';
 import * as React from 'react';
-import { renderWithTheme } from '../../../helpers/testUtils';
 import { Alert, AlertProps } from '../Alert';
 
-it('should render basic alert', () => {
-  ['error', 'warning', 'success', 'info', 'loading'].forEach(variant => {
-    expect(render({ variant })).toMatchSnapshot();
+it('verification of all variants of alert', () => {
+  const variants: AlertProps['variant'][] = ['error', 'warning', 'success', 'info', 'loading'];
+  variants.forEach(variant => {
+    const wrapper = shallowRender({ variant });
+    expect(wrapper.prop('variantInfo')).toMatchSnapshot();
   });
 });
 
 it('should render inline alert', () => {
-  expect(render({ display: 'inline' })).toMatchSnapshot();
+  expect(shallowRender({ display: 'inline' }).render()).toMatchSnapshot();
 });
 
 it('should render banner alert', () => {
```

Comment:
Although I understand why this test is interesting, I think it is redundant with the snapshot one. As there a css class linked to this banner mode, the snapshot already does tests it. Also, you don't test the inline mode, which is basically the same thing. The only purpose of those 2 flags is to adjust css which appears in the snapshot. Therefore they are tested IMO. WDYT ?